code_folder/C0_data_processing.py

```diff
@@ -57,27 +57,79 @@ def simulation_settings(simulation_settings):
 
     def process_all_assets(dict_values):
         # read timeseries with filename provided for technical parameters (efficiency and minimum and maximum storage level)
+
         for asset in dict_values["energyConversion"]:
             dict_asset = dict_values["energyConversion"][asset]
-            if "file_name" in dict_asset["efficiency"]:
+            # in case there is one value provided (case one input bus and one output bus)
```

Comment:
```suggestion
            # in case there is only one input bus and one output bus
```
"value" can be confused with dictionary pair {"value": ..., "unit":...}